contribs/gmf/src/directives/partials/search.html

```diff
@@ -8,4 +8,5 @@
 <span class="clear-button ng-hide"
         ng-hide="!ctrl.clearButton || ctrl.input_value == ''"
         ng-click="ctrl.onClearButton()"></span>
+  <input data-ng-model="color" data-ng-change="ctrl.setStyleColor(color)"></input>
```

Comment:
@adube Can you explain me what is that ? That creates a field below the search field that seems not used.
Is that an error ? What's the aim ?